src/Resources/public/core.js

```diff
@@ -941,20 +941,24 @@ var Backend =
 				alert('Could not find the SimpleModal frame');
 				return;
 			}
-			ul = frm.document.getElementById(opt.id);
+			ul = frm.document.getElementById('tl_select');
```

Comment:
This change broke an existing URL picker (similar to the old one from `tl_content`) in my test. Without this change the picker works as expected, but with it the `{{link_url::*}}` insert tag around the ID is missing.